packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanel.tsx

```diff
@@ -210,6 +183,27 @@ export function TransferPanel() {
     connectionState
   })
 
+  const isWarningToken = useMemo(() => {
+    if (!selectedToken) {
+      return false
+    }
+    return !!warningTokens[selectedToken.address.toLowerCase()]
+  }, [selectedToken, warningTokens])
+
+  const showWarningTokenToast = useCallback(() => {
+    const warningToken =
+      selectedToken && warningTokens[selectedToken.address.toLowerCase()]
+
+    if (!warningToken) {
+      return
+    }
+
+    const description = getWarningTokenDescription(warningToken.type)
+    warningToast(
+      `${selectedToken?.address} is ${description}; it will likely have unusual behavior when deployed as as standard token to Arbitrum. It is not recommended that you deploy it. (See ${DOCS_DOMAIN}/for-devs/concepts/token-bridge/token-bridge-erc20 for more info.)`
```

Comment:
Don't need optional chaining here or could even use `warningToken.address` which is probably more semantic